src/PerpetualPositionRouter.sol

```diff
@@ -36,6 +36,16 @@ contract PerpetualPositionRouter {
   // TODO: Should we deposit eth into the perpetual vault?
   receive() external payable {}
 
+  function combineArgs(uint8 funcName, uint160 sqrtPriceLimitX96) external pure returns (uint168) {
+    return (uint168(funcName) << 160) | uint168(sqrtPriceLimitX96);
+  }
```

Comment:
Also wondering about this

Reply:
yea I doubt anyone using this is going to do encoding on-chain or wait for an RPC call to encode, so let's move this to the test contract

Reply:
True it would be a very expensive way to do things.

https://github.com/ScopeLift/optimizoors-perpetual-protocol/pull/13/commits/164892848c8a48874a4e6cbc1a727b688a030482

Reply:
LGTM, one nit that internal functions in test contracts don't need a leading underscore. Rationale is that:
- All fns in test contracts are internal except for actual tests, and those start with `test`, so the leading underscore doesn't add meaning, just noise
- There's a ton of fns we already use that are internal without a leading underscore, i.e. all the ones from forge-std such as `assertEq` and `bound`

Reply:
Ah okay, I will change that and merge.